services/users.service.js

```diff
@@ -136,6 +136,33 @@ module.exports = {
 			},
 		},
 
+    getUserProfile: {
+      rest: "GET /me", // /api/users/me/
+      async handler(ctx) {
+        const { user } = ctx.meta;
+        
+        if (!user) {
+          throw new TirError("Missing x-access-token header variableeee", 401, "UNAUTHORIZED");
+        }
```

Comment:
Mivel ez egy védett útvonal, ezért nincs szükség külön csekkolni a` ctx.meta.user` értékét, ha nem tudja beállítani a framework, akkor el se jut idáig.

Reply:
Igaz